lib/util/node-utils-mock.js

```diff
@@ -0,0 +1,4 @@
+// In browser environments, this function is a no-op.
+export function nodeFilenameToStream( filename ) {
+	return filename;
```

Comment:
maybe we should return a mock node fs stream that's marked with `_readableState = false`? Returning the "wrong" type here feels pretty fragile.

Could we just return `undefined`?

Reply:
i agree with this. 
question: do we ever use `wpcom.media.addFiles` from the browser? If its only node then a much cleaner solution here would be to only add that function to wpcom in the case of a node build. 

I'll submit another patch now

Reply:
so `addFiles` is used in the browser! 
instead of the identity function, I made this throw an error.